app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -955,14 +955,18 @@
 					"earliest": "-48h",
 					"latest": "now"
 				},
-				"query": "index=redhatinsights AND  application=vulnerability\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval  has_rule='payload. has_rule'\n  | table _time,  has_rule\n  | where has_rule=True\n  | timechart span=24h aligntime=latest count( has_rule) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+				"query": "index=redhatinsights AND  application=vulnerability\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval  has_rule='payload.has_rule'\n  | table _time,  has_rule\n  | where has_rule=True\n  | timechart span=24h aligntime=latest count( has_rule) as count | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
 			},
 			"name": "search_cve_with_security_rules_24hrs"
 		},
 		"ds_ZlqEH4oo_ds_HRRsnn9w": {
 			"type": "ds.search",
 			"options": {
-				"query": "index=redhatinsights AND application=vulnerability\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval  has_rule='payload. has_rule'\n  | table _time,  has_rule\n  | where now()>relative_time(_time, \"+30d\")\n  | where has_rule=True\n  | timechart span=30d aligntime=latest count(has_rule) as count \n  | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]"
+				"query": "index=redhatinsights AND application=vulnerability\n  | spath \n  | spath path=events{} output=events\n  | stats by _time, events, application, event_type, account_id, context.display_name\n  | mvexpand events \n  | eval _raw=events\n  | kv\n  | eval  has_rule='payload.has_rule'\n  | table _time,  has_rule\n  | where now()>relative_time(_time, \"+30d\")\n  | where has_rule=\"true\"\n  | timechart span=30d aligntime=latest count(has_rule) as count \n  | appendpipe [| stats count | where count=0 | addinfo | eval time=info_min_time.\" \".info_max_time | makemv time | mvexpand time | table time count | rename time as _time ]",
+				"queryParameters": {
+					"earliest": "-30d@d",
```

Comment:
Shouldn't this be `-30d` instead of `-30@d` which aligns by day? How is it set elsewhere?

Reply:
This line was created by Splunk when I create the new object, and select `Last 30 days` for timestamp.

Reply:
Please check how we have it set on the other dashboard components.